models/articles-models.js

```diff
@@ -15,3 +15,28 @@ exports.fetchArticleById = (article_id) => {
       }
     });
 };
+
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `
+    SELECT COUNT(comments.article_id)::INT AS comment_count , 
+    articles.author, title, articles.article_id, topic,
+    articles.created_at, articles.votes, article_img_url FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC;
+    `
+    )
+    .then((result) => {
+      if (result.rows.length > 0) {
+        const articlesArr = result.rows;
+        return articlesArr;
+      } else {
+        return Promise.reject({
```

Comment:
I don't feel like we need this as we know that the articles have been inserted into the database. I would remove this as PSQL would through you an error similar to 

'relation "articles" does not exist" and would more than likely crash your server.